nixos/modules/services/networking/tailscale.nix

```diff
@@ -74,11 +74,10 @@ in {
     systemd.services.tailscaled = {
       wantedBy = [ "multi-user.target" ];
       path = [
-        config.networking.resolvconf.package # for configuring DNS in some configs
-        pkgs.procps     # for collecting running services (opt-in feature)
-        pkgs.getent     # for `getent` to look up user shells
-        pkgs.kmod       # required to pass tailscale's v6nat check
```

Comment:
Why not keep the formatting for these lines?

Reply:
Good question, I removed it again